runtimes/alibi-explain/tests/conftest.py

```diff
@@ -203,6 +203,70 @@ async def integrated_gradients_runtime() -> AlibiExplainRuntime:
     return rt
 
 
+@pytest.yield_fixture
+async def dummy_alibi_explain_client(tmp_path, settings) -> TestClient:
```

Comment:
nit: would it make sense to try to split this fixture into smaller ones?

Reply:
created a build rest server helper function for now as we do not use the inner objects outside of the function.